packages/cache/src/stylis-plugins.js

```diff
@@ -89,8 +89,9 @@ export let compat = element => {
   if (
     element.type !== 'rule' ||
     !element.parent ||
-    // .length indicates if this rule contains pseudo or not
-    !element.length
+    // positive .length indicates that this rule contains pseudo
+    // negative .length indicates that this rule has been already prefixed
+    element.length < 1
```

Comment:
this fixes prefixed rules (like `.css-hash::-webkit-input-placeholder {}`) - with the latest `stylis@4.0.11` and `stylis@4.0.12` those were replaced with their non-prefixed versions

Reply:
Is stylis's prefixing of `::-webkit-input-placeholder` incorrect as it stands?

Reply:
No, its prefixing it alright - we were screwing up the prefixed rules here though, before this change